### PR TITLE
Cfp

### LINKED ISSUE #5: CFP Page content
_HEADER_

# PWA Summit
### October 6-7, Online

_HERO_

[🎤 Be a speaker](url)
[📅 Reserve a date](url)

_MAIN_

# What is PWA Summit?

PWA Summit 2021 is a two-days single-track conference dedicated to all sides and colors of Progressive Web Apps. Each talk will consist of a 20 minute pre-recorded session and a live Q&A session following the talk. We're looking to draw an audience of web developers and decision makers who are interested in building and deploying Progressive Web Apps.

# Why to attend

## Learn

Learn everything about Progressive Web Apps, including development tips, tooling, libraries, debugging to browser support and business cases.

## Meet

Meet a diverse set of speakers from the PWA ecosystem representing multiple roles, skills, and experience levels.

## Build

After the Summit, you will have all the knowledge and skills to build or improve your web application

## Join

Join a huge community of like-minded developers in the interactive parts of PWA Summit

# Why to speak

* Share your findings, thoughts, experience with PWA in front of the thousands of the developers interested in this topic
* Selected speakers will be provided with recording guidance and support
* We will offer presentation coaching for any speaker who would like it
* New speakers are not only welcome, but encouraged to apply!
* An honorarium will be offered to all selected speakers wherever local regulations allow

# Organizing team

* Person 1
* Person 2
* ...

# Sponsors

* Google
* Microsoft
* Intel
* Samsung Internet
* Netlify

_FOOTER_

# © 2021 PWA Summit Team

* Code of Conduct
* @PWASummit Twitter
* Contact Us

### PR DESCRIPTION
**Pull request by Snugug**
Add landing page an Code of Conduct

Resolves #5 

[Staging Preview](https://deploy-preview-4--peaceful-darwin-86c101.netlify.app/)

### CODE REVIEW

**Review by webmaxru — CHANGES_REQUESTED** [review]

File: vite.config.js [hunk 1]
```diff
@@ -11,5 +12,66 @@ module.exports = defineConfig({
...
-  plugins: [eleventyPlugin()],
+  plugins: [
+    eleventyPlugin(),
+    pwa({
+      strategies: 'injectManifest',
+      srcDir: '',
+      filename: 'sw.js',
+      manifest: {
+        name: 'PWA Summit',
+        background_color: '#108484',
```
[the remark this answers is not in the document]
Comment (Snugug) [reply vite.config.js:23]: I'm not sure I understand this comment? `color` isn't a manifest property and, [per the spec](https://w3c.github.io/manifest/#background_color-member), is meant to describe the expected background color of the site (which currently is this teal) to provide a smooth transition to that background color when the PWA window loads..

File: vite.config.js [hunk 2]
```diff
@@ -11,5 +12,66 @@ module.exports = defineConfig({
...
   },
-  plugins: [eleventyPlugin()],
+  plugins: [
+    eleventyPlugin(),
+    pwa({
+      strategies: 'injectManifest',
+      srcDir: '',
+      filename: 'sw.js',
+      manifest: {
+        name: 'PWA Summit',
```
[the remark this answers is not in the document]
Comment (Snugug) [reply vite.config.js:22]: Thanks for pointing out that the plugin adds a `short_name` here that we probably don't want. I think in our case, having the short name and long name be the same is the better choice so I'm going to do that, as the `name` property is meant to be how the app is typically displayed to users (like in application lists or as icon labels) and serves as the accessible name, so we shouldn't overload it with descriptive information.

[the remark this answers is not in the document]
Comment (webmaxru) [reply vite.config.js:22]: For me, it's better to have `name` a bit more descriptive than `short_name` (it exists in spec to provide a _full_ name for the app/project) but it's not a big deal for this initial version so I approve your version.

File: vite.config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (webmaxru) [reply vite.config.js:26]: Thanks for the fix - yes, for sure I meant `theme_color `. And either from the false memory or from some early specs I thought that the splashscreen will contain app `name` printed using `theme_color` on the `background_color`. But I see that now it's one of the `icons` on the `background_color`.

File: templates/_layouts/wrapper.njk [hunk 3]
```diff
@@ -1,18 +1,143 @@
...
+    <footer class="banner--FOOTER">
+      <div class="container">
+        <nav class="nav">
+          {{ collections.all | eleventyNavigation('Main') | eleventyNavigationToHtml | safe }}
+        </nav>
+        <div class="footer">
+          <p><a href="https://twitter.com/PwaSummit" target="_blank">@PWASummit</a></p>
+          <p>
+            Organized by <strong>Google</strong>, <strong>Microsoft</strong>,
+            <strong>Samsung</strong>, <strong>Intel</strong>
```
Comment (webmaxru) [inline templates/_layouts/wrapper.njk:137]: Name to be updated to Samsung Internet
```suggestion
            <strong>Samsung Internet</strong>, <strong>Intel</strong>
```

**Review by webmaxru — CHANGES_REQUESTED** [review]

**Review by webmaxru — APPROVED** [review]

**Pull request merged**